### PR TITLE
chore: add storybook to website deployment, 3.0 publish core cleanup

### PR DESCRIPTION
**Pull request by coryrylan**
Signed-off-by: Cory Rylan <crylan@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [x] Docs have been added / updated (for bug fixes / features)
* [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?
- Adds storybook demo of Core to https://clarity.design/storybook/core
- Fixes publish artifacts for core (missing metadata for entrypoints)
- Improve component render performance using CSS contain
- Remove unused demo code

* [x] Bugfix
* [ ] Feature
* [ ] Code style update (formatting, local variables)
* [ ] Refactoring (no functional changes, no api changes)
* [x] Build related changes
* [x] CI related changes
* [ ] Documentation content changes
* [x] clarity.design website / infrastructure changes
* [ ] Other... Please describe:

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

### CODE REVIEW

File: src/clr-core/welcome.stories.ts [hunk 1]
```diff
@@ -62,6 +62,8 @@ export const gettingStarted = () => {
 
       <p>
         Clarity Core is a Web Component implementation of the <a href="https://clarity.design">Clarity Design System</a>.
+        Clarity Core provides a set of reusable UI components that work in any JavaScript framework or no framework at all.
+        You can read more about Core and its goals in our <a href="https://medium.com/claritydesignsystem/clarity-core-72f6d3a029bc">Medium blog post</a>.
```
Comment (coryrylan) [inline src/clr-core/welcome.stories.ts]: TODO: going to add a comment that the initial batch of components we will be treating as Beta but feel they are stable enough for teams to start trying and work with getting feedback on.

File: src/clr-core/welcome.stories.ts [hunk 2]
```diff
@@ -62,6 +62,12 @@ export const gettingStarted = () => {
 
       <p>
         Clarity Core is a Web Component implementation of the <a href="https://clarity.design">Clarity Design System</a>.
+        Clarity Core provides a set of reusable UI components that work in any JavaScript framework or no framework at all.
```
Comment (gnomeontherun) [inline src/clr-core/welcome.stories.ts]: I was hoping we'd see more content on the landing page, or just remove the current landing page and go straight to 'get started'

Comment (coryrylan) [reply src/clr-core/welcome.stories.ts]: Makes sense, I can update that

Comment (coryrylan) [reply src/clr-core/welcome.stories.ts]: Updated, added content to landing page as well as new browser support and improved framework docs. https://5e56e7bd7976455cb75d5a51--vmware-clarity.netlify.com/storybook/core

**Pull request merged**